CHANGELOG.md

```diff
@@ -15,12 +15,15 @@
 - PR #6125 Add support for `Series.mode` and `DataFrame.mode`
 - PR #6271 Add support to deep-copy struct columns from struct column-view
 - PR #6262 Add nth_element series aggregation with null handling
+- PR #6318 Add support for reading Struct and map types from Parquet files
 - PR #6316 Add StructColumn to Python API
 - PR #6247 Add `minmax` reduction function
 - PR #6232 `Json` and `Avro` benchmarking in python
 - PR #6139 Add column conversion to big endian byte list.
 - PR #6220 Add `list_topics()` to supply list of underlying Kafka connection topics
 - PR #6254 Add `cudf::make_dictionary_from_scalar` factory function
+- PR #6262 Add nth_element series aggregation with null handling
```

Comment:
Bad merge?

Reply:
Could be.  Was auto-merged I believe.  I'll fix.